public/index.html

```diff
@@ -9,7 +9,9 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <!--
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
- Delete comments in the code. Also, delete the `./public/favicon.ico` file. This is a react logo. You don't need it.